astropylibrarian/reducers/utils.py

```diff
@@ -99,6 +99,7 @@ def iter_sphinx_sections(
             )
         else:
             if content_callback:
+                print('cleaning content')
```

Comment:
Fine to not include in this PR, but may want to use `logging` / a custom logger throughout the package

Reply:
Good call. I've adding a logging set up. In this case, I also deleted this print/log line because it's not terribly useful anymore!